vertical-pod-autoscaler/pkg/recommender/input/cluster_feeder.go

```diff
@@ -255,48 +255,48 @@ func (feeder *clusterStateFeeder) InitFromCheckpoints() {
 	}
 
 	for namespace := range namespaces {
-		klog.V(3).Infof("Fetching checkpoints from namespace %s", namespace)
+		klog.V(3).InfoS("Fetching checkpoints", "namespace", namespace)
 		checkpointList, err := feeder.vpaCheckpointClient.VerticalPodAutoscalerCheckpoints(namespace).List(context.TODO(), metav1.ListOptions{})
 		if err != nil {
-			klog.Errorf("Cannot list VPA checkpoints from namespace %s. Reason: %+v", namespace, err)
+			klog.ErrorS(err, "Cannot list VPA checkpoints", "namespace", namespace)
 		}
 		for _, checkpoint := range checkpointList.Items {
 
-			klog.V(3).Infof("Loading VPA %s/%s checkpoint for %s", checkpoint.ObjectMeta.Namespace, checkpoint.Spec.VPAObjectName, checkpoint.Spec.ContainerName)
+			klog.V(3).InfoS("Loading VPA checkpoint", "namespace", checkpoint.ObjectMeta.Namespace, "VPA", checkpoint.Spec.VPAObjectName, "container", checkpoint.Spec.ContainerName)
```

Comment:
nit: consistency with other log lines where it's lowercase vpa
```suggestion
			klog.V(3).InfoS("Loading VPA checkpoint", "namespace", checkpoint.ObjectMeta.Namespace, "vpa", checkpoint.Spec.VPAObjectName, "container", checkpoint.Spec.ContainerName)
```